apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -239,38 +272,84 @@ protected Optional<TableMetadata> mapTableResponseToTableMetadata(
 
     TableMetadata.TableMetadataBuilder<?, ?> builder =
         TableMetadata.builder()
-            .creator(tableResponseBody.getTableCreator())
-            .dbName(tableResponseBody.getDatabaseId())
-            .tableName(tableResponseBody.getTableId())
+            .creator(Objects.requireNonNull(tableResponseBody.getTableCreator()))
+            .dbName(Objects.requireNonNull(tableResponseBody.getDatabaseId()))
+            .tableName(Objects.requireNonNull(tableResponseBody.getTableId()))
             .isPrimary(
                 tableResponseBody.getTableType()
                     == GetTableResponseBody.TableTypeEnum.PRIMARY_TABLE)
             .isTimePartitioned(tableResponseBody.getTimePartitioning() != null)
             .isClustered(tableResponseBody.getClustering() != null)
-            .retentionConfig(getTableRetention(tableResponseBody).orElse(null));
-    if (tableResponseBody.getCreationTime() != null) {
-      builder.creationTimeMs(tableResponseBody.getCreationTime());
-    }
+            .retentionConfig(getTableRetention(tableResponseBody).orElse(null))
+            .jobExecutionProperties(getJobExecutionProperties(tableResponseBody));
```

Comment:
It comes from a concurrently added https://github.com/linkedin/openhouse/pull/216, now it's part of the metadata, in that PR execution props were fetched separately, the reason why this change is not in that PR is because of development in parallel.